plugins/gatsby-plugin-md-from-sponsors-csv/gatsby-node.js

```diff
@@ -0,0 +1,24 @@
+exports.sourceNodes = async ({ actions, createContentDigest, createNodeId, getNodesByType }) => {
+  const { createNode, createParentChildLink } = actions;
+
+  const sponsors = getNodesByType('SponsorsCsv');
```

Comment:
tu as bien récupéré le bon type, c'est top.

en fait dans le pugin, sans le dissocier en deux, on pourrait juste avoir deux lignes qui récupèrent les 2 types et faire une boucle
- soit en séparant le type
- soit en mettant un type unique `markdownDescription` vu que de toute façon on les récupère via le lien parent enfant (on aura toujours la bonne description markdwon associé au parent à partir duquel on a créé le noeud markdwon)

Reply:
Bien vu! Merchi. Je creuse ça.

Reply:
Autre question : l'identifiant :
```javascript
const childrenNodeId = createNodeId(`HackavizParticipantsCsv-${index}-MarkdownDescription`);
``` 
c'est juste un identifiant associé à chaque élément pour éviter ce genre de warning ?
https://reactjs.org/docs/reconciliation.html#recursing-on-children

Reply:
c'est côté Gatbsy : il faut que chaque noeud ait un identifiant unique, et différent de ce que tu pointes.

côté React, quand tu itères, il faut aussi que les éléments aient des identifiants unique (qui ne sont pas des index de tableau) pour pouvoir repérer ce qui a changé et rerendre uniquement ces éléments là.

Reply:
du coup le code du plugin pourrait ressembler à 

```
const sponsors = getNodesByType('SponsorsCsv');
const participants = getNodesByType('HackavizParticipantsCsv');

[...sponsors, ...participants]..forEach((parentNode, index) => {
})
    const childrenNodeId = ...
    const markdownNode = ..
    createNode(markdownNode);
    createParentChildLink({ parent: parentNode, child: markdownNode });
```